geemusic/__init__.py

```diff
@@ -1,14 +1,15 @@
 from flask import Flask
 from flask_ask import Ask
 
-from utils.music_queue import MusicQueue
 from utils.music import GMusicWrapper
 
 app = Flask(__name__)
 ask = Ask(app, '/alexa')
 
-queue = MusicQueue()
 api = GMusicWrapper.generate_api(logger=app.logger)
 
+from utils.music_queue import MusicQueue
+
+queue = MusicQueue()
```

Comment:
Yep, it allows it to import the instance of GMusicWrapper without an error from it trying to load QueueManager before QueueManager is loaded.